packages/dita-example-sx-modules-xsd-reference-mod/src/install.js

```diff
@@ -29,13 +29,13 @@ export default function install() {
 
 			var isFirstCell = true;
 			stepData.childNodeStructure = [stepData.rowNodeName];
-
+			// TODO concats 
 			stepData.columns.forEach(function(column) {
 				var selector = 'child::*/' + column.currentNodeName;
 				selector += column.otherNodeNames
 					? ' or child::*/' + column.otherNodeNames.join(' or child::*/')
 					: '';
-				if (!evaluateXPathToBoolean(selector, tableNode, readOnlyBlueprint)) {
+				if (!evaluateXPathToBoolean(xq(selector), tableNode, readOnlyBlueprint)) {
```

Comment:
I think this XPath can be constructed differently that removed the use of `xq()`: by using string interpolation and the `name()` function this can be written a bit more natural. In general for code that is not _that_ performance-critical (sx config is EXTREMELY performance critical) I do not advocate generating XPaths by hand: just use functions and variables

Reply:
How would you suggest to do that?

Reply:
Something like this?

```
	if (!evaluateXPathToBoolean(xq`child::*/child::*[name() = ${column.currentNodeName}]`, tableNode, readOnlyBlueprint)) {
		return;
	}

	if (column.otherNodeNames && column.otherNodeNames.some(nodeName => evaluateXPathToBoolean(xq`child::*/child::*[name() = ${nodeName}]`, tableNode, readOnlyBlueprint)) {
		return;
	}
```

Reply:
This has the added benefit of being more readable, more robust against nonsensical input (`currentNodeName` containing slashes and such) and just better

Reply:
+1

This is actually one of the main reasons we invented the `xq` API, so updating the `dita-example-editor` correctly here is the API's "poster child" :D